### PR TITLE
Review based updates

### PR DESCRIPTION
**Pull request by shemnon**
**Description**:
- Change value written to a google protobuf wrapper type
- Clarified that the stateChanges field is repeated.
- Clarified that all contract accounts state changes are recorded.
- Added references for external tracing standards
- Fix a few typos

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-260.md [hunk 1]
```diff
@@ -53,9 +55,11 @@ transactions.
 
 ### Add ContractStorageChanges to ContractFunctionResult
 
-Add and populate a new field `stateChanges` of type `ContractStorageChagnes` to
-the `ContractFunctionResult` message. This will be populated whenever a
-transaction causes contract storage states to change.
+Add and populate a new repeated field `stateChanges` of
+type `ContractStorageChanges` to the `ContractFunctionResult` message. This will
```
Comment (steven-sheehy) [inline HIP/hip-260.md:59]: Can we show this protobuf snippet instead of just describing it?

File: HIP/hip-260.md [hunk 2]
```diff
@@ -198,6 +204,9 @@ not sustainable for further scaling efforts.
 
 ## References
 
+- [OpenEtereum Trace Module](https://openethereum.github.io/JSONRPC-trace-module)
```
Comment (steven-sheehy) [inline HIP/hip-260.md:207]: `OpenEthereum`

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: HIP/hip-260.md (hunk 1 above)
Comment (shemnon) [reply HIP/hip-260.md:59]: done

File: HIP/hip-260.md (hunk 2 above)
Comment (shemnon) [reply HIP/hip-260.md]: done

**Review by SimiHunjan — APPROVED** [review]

File: HIP/hip-260.md (hunk 1 above)
Comment (shemnon) [reply HIP/hip-260.md]: fixed, set to `ContractStateChange` in header and text

**Review by sergmetelin — APPROVED** [review]

**Review by steven-sheehy — APPROVED** [review]

**Review by sergmetelin — APPROVED** [review]

**Pull request merged**